src/Mod/Arch/ArchWall.py

```diff
@@ -1071,9 +1167,7 @@ def attach(self,vobj):
         self.Object = vobj.Object
         from pivy import coin
         tex = coin.SoTexture2()
-        image = Draft.loadTexture(Draft.svgpatterns()['simple'][1], 128)
-        if not image is None:
-            tex.image = image
+        tex.image = Draft.loadTexture(Draft.svgpatterns()['simple'][1], 128)
```

Comment:
Werner added a test here to check that `tex.image` is not `None` because if it is `None` it will crash the program. As `Draft.loadTexture` may return `None` if the texture is not found, you should keep the test.

```python
    image = Draft.loadTexture(Draft.svgpatterns()['simple'][1], 128)
        if image is not None:
            tex.image = image
```

Reply:
Hmmm... just download the code on 23.11.2019, not aware there is some update, and thought git / github would somehow alert the 'conflict'?